src/component/pages/MyProfile.js

```diff
@@ -0,0 +1,6 @@
+const MyProfile = () => (
+  <div>
+    <h3>profile</h3>
+  </div>
+);
+export default MyProfile;
```

Comment:
- [ ] The profile page is blank. Kindly make sure to implement the required feature for the profile page:
- - [ ] Composes of two layouts and list ONLY the rockets reserved and missions joined by the user (as per design).
- - [ ] Renders a list of all joined missions that should be done filtering the state for each module.
![image](https://user-images.githubusercontent.com/33767951/147229837-455ca622-941a-41de-9781-1a3d7b33601d.png)